packages/design-system/src/components/web-components/ds-date-field/ds-date-field.tsx

```diff
@@ -0,0 +1,100 @@
+import { define } from '../preactement/define';
+import { SingleInputDateFieldProps } from '../../DateField/SingleInputDateField';
+import { SingleInputDateField } from '../../DateField';
+import { parseBooleanAttr, parseDateAttr, parseIntegerAttr } from '../wrapperUtils';
+
+const attributes = [
+  'name',
+  'root-id',
+  'inversed',
+  'value',
+  'default-month',
+  'from-date',
+  'from-month',
+  'from-year',
+  'to-date',
+  'to-month',
+  'to-year',
+  'hint',
+  'label',
+  'requirement-label',
+  'error-message',
```

Comment:
I was reviewing your `argTypes` additions, and I realized that it's missing the standard set of form-field attributes [like we include here](https://github.com/CMSgov/design-system/blob/tamara/add-ds-date-field-web-component/packages/design-system/src/components/web-components/ds-text-field/ds-text-field.tsx#L25). It's a little helper we added to reduce duplication. We don't actually have the same sort of thing for the `argTypes` definitions, though; we've just been copying and pasting those from other web components and changing the descriptions if needed. You'll see in that `formAttrs` array that there are a few attributes missing from this list.

Reply:
@pwolfert - Thanks for the feedback. I've added formAttrs to the attribute list and copied the argType definitions for the missing form-related attributes from other web components that have them. Is there anything else I need to include in this list? I noticed that the `SingleInputDateField` controls include things like `size` and `rows`, although it's unclear to me what these do. Could you provide some guidance on whether these are necessary?

Reply:
The new controls look good to me. The `size` and `rows` shouldn't really be on there, but the controls on the React stories are generated mostly by Storybook, which is going off of the TypeScript types. We might be able to remove it by explicitly removing those attributes from the prop types of `SingleInputDateField` like by using `Omit` [on this line](https://github.com/CMSgov/design-system/blob/tamara/add-ds-date-field-web-component/packages/design-system/src/components/DateField/SingleInputDateField.tsx#L104)